### DISCUSSION TITLE
Moved to sveltekit?

### DISCUSSION THREAD

**Discussion by ctwhome** [saadeghi/daisyui#460] [General]
wow that is a bold move. I am also getting double feelings with Nuxt for a long time. what have been your reasons to make the change?

**Comment by saadeghi** [comment]
On daisyUI document website, I need more than 120 PostCSS files to be used on development environment because I don't want to build the daisyUI Node package to see the updates every time I change one line of style.  

- Build time on Nuxt on was slow.  
A normal website with 1 CSS file is fast but more imports, significantly slows the build time.
- Nuxt is a dependency hell 
They have wrapper modules for everything. You need additional dependencies for Post CSS 8, Tailwind CSS, etc... to avoid incompatibilities but each dependency just adds additional bugs and limitations and prevents you from using the latest version of original package.
- Countless bugs and slow development.  
There's still an open issue from 2020 on supporting PostCSS 8. I don't know how many more years I should wait front that.
- Developer experience
Svelte and Sveltekit have a much better developer experience than Vue and Nuxt. Sveltekit is still in early development and it's not even v1.0 but it's 100x more stable and trustworthy than Nuxt.

**Reply by AshSimmonds** [reply]
Well I didn't need a reason to learn another toolkit, but I guess I do now, so thanks - ish.

**Reply by ctwhome** [reply]
Thanks for your comments!

I share your pain points. My problem is that I have been advocating nuxt for the last few years, it feels wrong to sudently move. But nuxt 3 is making things a bit hard for me, waiting for it for ages, and also  I don't agree with some decisions they made, like creating nuxt bridge rather than focus on nuxt 3 from the begining, slowing both versions IMO. 
Modules are good when they are release but get outdated or deprecated very rapidly. Nuxt tailwindcss was in version 2 for at least 6 months after version 3 came out. And nuxt content was deprecated within the first year of life. If you install a nuxt module you depend on the maintainer to update latest versions of the packages and that rarely happens, or it is very slow. No better off not using any modules at all. 

Another point is that the roadmap has also been keept very secretly and it gave me a lot of stress since last year. 

I was starting to think to try Sveltekit recently, but this bold move you did and the reasons are enough to start taking a look more seriously at it.

**Reply by vnphanquang** [reply]
I am biased because I have used Svelte much more than i used Vue. But the developer experience is absolutely incredible. This move to svelte kit actually helped me want to contribute more than I did before. Great job!
